Sources/Account/AddressHash.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+import Foundation
+import LibMobileCoin
+
+struct AddressHash {
+    let data16: Data16
+    
+    var hexBytes: String {
```

Comment:
Minor nit, but the name 'hexBytes' makes it seem like this value is raw data bytes in some form, versus a string... can we rename to hexString?

Reply:
good point, updated